jenkins-job-builder/config/definitions/jjb.yml

```diff
@@ -1,6 +1,6 @@
 - job:
     name: jenkins-job-builder
-    node: (small && trusty) || master
+    node: small && trusty
```

Comment:
that would potentially give us wheezy nodes though, unless we changed mita's config to remove wheezy